components/org.wso2.ei.analytics.elk/pom.xml

```diff
@@ -95,6 +76,11 @@
         </dependency>
     </dependencies>
 
+    <properties>
+        <elk.httpclient.version>4.5.8</elk.httpclient.version>
+        <elk.httpcore.version>4.4.11</elk.httpcore.version>
+    </properties>
```

Comment:
move version properties to root pom

Reply:
Fixed in a6b950256f312e59016a853e11d625a06ba07ae0